sdk/go/gcp/workstations/workstationCluster.go

```diff
@@ -171,7 +177,7 @@ import (
 //				Annotations: pulumi.StringMap{
 //					"label-one": pulumi.String("value-one"),
 //				},
-//			}, pulumi.Provider(google_beta))
```

Comment:
Why did we used to specify an explicit provider, but we don't anymore?

Reply:
I'm not sure but this looks like a fix. The variable used to be unbound. I think the change is for good.